src/applications/vaos/new-appointment/components/DateTimeSelectPage/index.jsx

```diff
@@ -94,10 +95,16 @@ ErrorMessage.propTypes = {
   history: PropTypes.object,
 };
 
-function goForward({ dispatch, data, history, setSubmitted }) {
+function goForward({
+  dispatch,
+  data,
+  history,
+  setSubmitted,
+  isAppointmentSelectionError,
+}) {
   setSubmitted(true);
 
-  if (data.selectedDates?.length) {
+  if (data.selectedDates?.length && !isAppointmentSelectionError) {
```

Comment:
Change to not allow the user to continue when there is an duplicate appointment error.